qemu/tests/win_virtio_driver_installer_repair.py

```diff
@@ -70,11 +70,12 @@ def run(test, params, env):
         test_func = "win_driver_installer_test.%s_test" % test_name
         driver_test_params = params.get('driver_test_params_%s'
                                         % driver_name, '{}')
+        if driver_name == "viofs":
+            virtio_fs_utils.run_viofs_service(test, params, session)
+
         if driver_name == "balloon":
             balloon_test_win = BallooningTestWin(test, params, env)
             driver_test_params = {"balloon_test_win": balloon_test_win}
-        elif driver_name == "viofs":
-            virtio_fs_utils.run_viofs_service(test, params, session)
```

Comment:
Previous method is actually better, an extra 'if' checking was wasting resource.

Reply:
Thanks Qianqian.
If using 'elif' for viofs, then viofs will have no "driver_test_params" defined, because it does not go to 'else' condition.